CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+Code of Conduct
+===============
+
+The following text is derived from the [Django Project's Code of
+Conduct](https://www.djangoproject.com/conduct/), which is licensed as
+[Creative Commons Attribution](https://creativecommons.org/licenses/by/3.0/).
+
+Like the technical community as a whole, the restic team and community is made
+up of a mixture of professionals and volunteers from all over the world,
+working on every aspect of the mission - including mentorship, teaching, and
+connecting people.
+
+Diversity is one of our huge strengths, but it can also lead to communication
+issues and unhappiness. To that end, we have a few ground rules that we ask
+people to adhere to. This code applies equally to founders, mentors and those
+seeking help and guidance.
+
+This isn't an exhaustive list of things that you can't do. Rather, take it in
+the spirit in which it's intended - a guide to make it easier to enrich all of
+us and the technical communities in which we participate.
```

Comment:
Does the above paragraph add substantial value to the document? I feel like it distracts from the rest and it isn't clear whether it suggests an increase or decrease in intent to enforce. "Isn't exhaustive of things that you can't do" implies that that it might encompass more than just written. "Guide to make it easier" suggests that it might be overly literal and simplistic, and is actually intended for reasonable interpretation.

Reply:
You're right, and it's even worse than that.  The first sentence is an open door to expelling anyone from the project for nearly any reason, leaving anyone's behavior open to interpretation according to whoever's doing the interpreting.  Proscribed behaviors don't have to be on the list to be forbidden.  Whoever's in power decides what's allowed, which also allows them to decide unfairly among those they like and dislike.

The second sentence is a rhetorical trap: how could anyone object to that?  Anyone who objects to it must be against the spirit of enriching the community.  Therefore their objections are invalid (and they should probably be expelled after the code is imposed, to protect and enrich the community, of course).

These illustrate why the wholesale adoption of pre-packaged codes of conduct is a very bad idea.  It's like a water fountain that's been set up with a sign that says, "Help yourself!  Be enriched!"  Seems like a great, generous idea--yet the water is poisoned, and those (i.e. projects) who drink of it become infected.

And how, then, would the infection be eradicated?  Who could ever *erase* a code of conduct?  Whoever would want to do that must be a *toxic* person who doesn't care about the health of the community.  Begone, evildoer.